public/src/reimbursement-policy.md

```diff
@@ -114,47 +128,51 @@ All dollar amounts discussed are in Canadian dollars.
     - Taxis
     - Ride-Share (e.g. Uber, Lyft)
+  - Paid parking at public or private lots
 
 ### Lodging
 
-- Reimbursement requests can be made for lodging **only when deemed necessary for the successful operation of a Speedcubing Canada event**.
-- No lodging expenses may be reimbursed beyond the night before the first day or the night after the final day of a Speedcubing Canada event.
+- Reimbursement requests are allowed to be made for lodging, but **must** only be approved when deemed absolutely necessary for the successful operation of Speedcubing Canada Supported Events.
+- Lodging expenses **must not** be reimbursed beyond the night before the first day or the night after the final day of a Speedcubing Canada Supported Event.
 - Expenses covered by this policy include:
   - Hotels
   - Temporary rentals (e.g. Airbnb)
 
 ## Submitting Allowed Expenses
 
-- To request reimbursement for allowed expenses, the individual requesting reimbursement should email [reimbursements@speedcubingcanada.org](mailto:reimbursements@speedcubingcanada.org).
-  - The email _should_ include "reimbursement" in the subject line.
-  - The email **must** include the name of the Speedcubing Canada event the allowed expenses are associated with.
+- To request reimbursement for allowed expense(s), the individual(s) and/or group(s) requesting reimbursement **must** email [reimbursements@speedcubingcanada.org](mailto:reimbursements@speedcubingcanada.org).
+  - The email _should_ include “reimbursement” in the subject line.
+  - The email **must** meet one of the two following requirements:
+    - Include the name(s) of the Speedcubing Canada Supported Event(s) the allowed expense(s) are associated with.
+    - If the allowed expense(s) are for equipment that will have ongoing use at Speedcubing Canada Supported Events, but cannot be directly associated with specific Speedcubing Canada Supported Events(s), provide written explanation to justify the equipment's benefit to Speedcubing Canada Supported Events in a particular region.
   - The email **must** include receipts attached to the email.
-    - No expense will be reimbursed without a receipt (with the exception of automobile allowance).
+    - An expense **must not** be reimbursed without a receipt (with the exception of automobile allowance).
   - For each receipted expense, the email **must** include which policy (e.g. Venues and associated costs, Equipment Type 1) the expense is covered by.
-- Reimbursements _should_ be submitted within 60 days of when the out-of-pocket payment was made and **must** be submitted within 90 days of when the out-of-pocket payment was made.
-  - Reimbursements submitted between 61 and 90 days of when the out-of-pocket payment was made **must** include a written explanation for the delay.
+- Reimbursements _should_ be submitted within 60 days of when the out-of-pocket payment was made and **must** be submitted within 365 days of when the out-of-pocket payment was made.
```

Comment:
I don't have a strong opinion on this, but a year feels excessive and potentially annoying. I think 180 days is more realistic. Perhaps we should have a quick poll of the officers?

Reply:
No one seems to have an opinion.